harvard-lts/CURIOSity
# More Arabic updates based on feedback

**More Arabic updates based on feedback**
* * *

# What does this Pull Request do?
More updates based on Johannes second round of feedback

# How should this be tested?

A description of what steps someone could take to:
* Spin up Curiosity locally from this branch and log in
* Add “Arabic” as a language to one of your collections (I chose the demo collection) and switch to Arabic to view the following changes:
   * Dropdown menu is aligned right
   * Sidenav menu links no longer have letter-spacing (I had only fixed the "selected" state before)
   * Sidenav subpages are indented by 20px

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@VanessaVenti @enriquediaz
